### DISCUSSION TITLE
Hola amigos, buen dia.

### DISCUSSION THREAD

**Discussion by Armando9576** [Launch-X-Latam/MisionFrontEnd#441] [Team Posts]
Tengo un detallito, a mí me dieron accesos el día jueves en la noche, así que me siento un poco perdido, podrían apoyarme diciéndome más o menos como es que está tomando camino el curso, cabe aclarar que estoy viendo las repeticiones entames, pero no termino de entender todo.

**Comment by gumodi35** [comment]
Que tal Armando! <br>
Mira las actividades que se realizaron estan el repositotio **MIsion Frontend**, hay viene acerca del ciclo de vida de software, la toma de requerimientos de una aplicacion, el ejercicio del Abogabot etc... <br>
El dia viernes fuen un vistazo de HTML, la sintaxis y las herramientas que se ocupan para escribir codigo. <br>
Si ya tienes acceso al Teams, dale una revisada a las seciones que se han publicado, hay esta lo que se ha hecho. <br>
![practica1](https://user-images.githubusercontent.com/80011796/194730233-9cc8a0d8-e97d-41ac-bff2-89d013a9331b.png)<br>
Aqui puedes leer los conceptos y ver que actividad realizar.
<br>
![practica2](https://user-images.githubusercontent.com/80011796/194730239-122f9c32-6c58-426c-a085-ced788e9ca67.png)<br>
No te estreses, tienes tiempo para ponerte al dia y lo mas importante, pues aprende lo que vas haciendo, cualquier duda manda mensaje! Saludos! :+1:

**Comment by LuisCarlosJaramillo** [comment]
Buenas noches, a mí me habilito el perfil en Github pero el link del
formulario que redirige al grupo de  Teams sale como si hubiese vencido

El sáb, 8 oct 2022 a la(s) 17:46, Gualberto ***@***.***)
escribió:

> Que tal Armando!
>
> Mira las actividades que se realizaron estan el repositotio *MIsion
> Frontend*, hay viene acerca del ciclo de vida de software, la toma de
> requerimientos de una aplicacion, el ejercicio del Abogabot etc...
>
> El dia viernes fuen un vistazo de HTML, la sintaxis y las herramientas que
> se ocupan para escribir codigo.
>
> Si ya tienes acceso al Teams, dale una revisada a las seciones que se han
> publicado, hay esta lo que se ha hecho.
>
> [image: practica1]
> <https://user-images.githubusercontent.com/80011796/194730233-9cc8a0d8-e97d-41ac-bff2-89d013a9331b.png>
>
> Aqui puedes leer los conceptos y ver que actividad realizar.
>
>
> [image: practica2]
> <https://user-images.githubusercontent.com/80011796/194730239-122f9c32-6c58-426c-a085-ced788e9ca67.png>
>
> No te estreses, tienes tiempo para ponerte al dia y lo mas importante,
> pues aprende lo que vas haciendo, cualquier duda manda mensaje! Saludos!
> 👍
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Launch-X-Latam/teams/mision-frontend/discussions/10/comments/1>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATSEAVLJTK5X2HPP3BHUKATWCH2VDANCNFSM6AAAAAARANJI2Q>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


--

**Comment by gumodi35** [comment]
puedes intentarlo nuevamente, en caso de que no te funcione, manda mensaje a contacto innovaccion y comentales, no estoy seguro si tiene vigencia. :+1:

**Comment by EJosueMenjivar** [comment]
Muchas gracias
